mypy/semanal.py

```diff
@@ -2210,8 +2210,8 @@ def check_no_global(self, n: str, ctx: Context,
                         is_func: bool = False) -> None:
         if n in self.globals:
             if is_func and isinstance(self.globals[n].node, FuncDef):
-                self.fail(("Name '{}' already defined (overload variants "
-                           "must be next to each other)").format(n), ctx)
+                self.fail(("Name '{}' already defined on line "
+                           "{}").format(n, self.globals[n].node.line), ctx)
```

Comment:
I removed this for now, instead using the regular code path for repeat definition. Referring to the previous line sounds like something that every repeat definition could benefit from.